src/bridge/AbsBridge.sol

```diff
@@ -127,6 +127,7 @@ abstract contract AbsBridge is Initializable, DelegateCallAware, IBridge {
         if (afterDelayedMessagesRead > 0) {
             delayedAcc = delayedInboxAccs[afterDelayedMessagesRead - 1];
         }
+        // @review should use accumulateSequencerInbox here
```

Comment:
this was not fixed in https://github.com/OffchainLabs/nitro-contracts/pull/160 yet

Reply:
I implemented your suggestion when both sync and delay proofs were implemented.

when removing the sync proof, the auxiliary helper functions in the Messages.sol library were no longer used and removed. Therefore there is no longer an accumulateSequencerInbox function and hence the bridge was left unchanged.